apps/example-app-vue/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "../../tsconfig.base.json",
```

Comment:
added extends to the tsconfig base from the root directory

Reply:
Do we need a tsconfig.json and tsconfig.base.json?   I thought eslint or stylelint only use tsconfig.json so it requires both, but I could be wrong